language/move-lang/stdlib/modules/libra_account.move

```diff
@@ -4,11 +4,13 @@ address 0x0:
 module LibraAccount {
     use 0x0::LibraCoin;
     use 0x0::Hash;
-    use 0x0::Event;
-    use 0x0::Transaction;
+    use 0x0::U64Util;
     use 0x0::AddressUtil;
+    use 0x0::BytearrayUtil;
+    use 0x0::LibraTransactionTimeout;
+    use 0x0::Transaction;
 
-    // Every Libra account has a LibraAccount::T resource
+    // Every Libra account has a LibraAccount.T resource
```

Comment:
This seems like it should be "LibraAccount::T"

Reply:
Several other places later in the file have the same thing, too.

Reply:
Good spots. For some reason my regex didn't match these. I'll go through and fix them all up.